test/e2e/cypress/integration/single-backend/users.spec.js

```diff
@@ -334,10 +326,7 @@ describe('Users', function() {
       .click()
       .type('{selectall}SuM UNnv4L33d JZOOOOO000n K0d')
 
-    cy.get('[data-cy="UserUpdate-customTab--dangerIcon"]').should(
-      'be',
-      'visible'
-    )
+    cy.get('[data-cy="UserUpdate-customTab--dangerIcon"]').should('be.visible')
```

Comment:
To be consistent with other tests ?
```suggestion
    cy.get('[data-cy="UserUpdate-customTab--dangerIcon"]').should('exist')
```

Reply:
As above.